MdeModulePkg/Library/SmmPerformanceLib/SmmPerformanceLib.c

```diff
@@ -7,58 +7,28 @@
   performance information.
 
   Copyright (c) 2011 - 2023, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.
 SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
 #include <Guid/PerformanceMeasurement.h>
 
-#include <Library/PerformanceLib.h>
 #include <Library/DebugLib.h>
-#include <Library/SmmServicesTableLib.h>
+#include <Library/MmServicesTableLib.h>
 #include <Library/PcdLib.h>
-#include <Library/BaseMemoryLib.h>
-
-//
-// The cached SMM Performance Protocol and SMM PerformanceEx Protocol interface.
-EDKII_PERFORMANCE_MEASUREMENT_PROTOCOL  *mPerformanceMeasurement = NULL;
-BOOLEAN                                 mPerformanceMeasurementEnabled;
-VOID                                    *mPerformanceLibExitBootServicesRegistration;
-
-/**
-  This is the Event call back function is triggered in SMM to notify the Library
-  the system is entering runtime phase.
-
-  @param[in] Protocol   Points to the protocol's unique identifier
-  @param[in] Interface  Points to the interface instance
-  @param[in] Handle     The handle on which the interface was installed
+#include <Library/PerformanceLib.h>
 
-  @retval EFI_SUCCESS SmmAtRuntimeCallBack runs successfully
- **/
-EFI_STATUS
-EFIAPI
-SmmPerformanceLibExitBootServicesCallback (
-  IN CONST EFI_GUID  *Protocol,
-  IN VOID            *Interface,
-  IN EFI_HANDLE      Handle
-  )
-{
-  //
-  // Disable performance measurement after ExitBootServices because
-  // 1. Performance measurement might impact SMI latency at runtime;
-  // 2. Performance log is copied to non SMRAM at ReadyToBoot so runtime performance
-  //    log is not useful.
-  //
-  mPerformanceMeasurementEnabled = FALSE;
+#include "SmmPerformanceLibInternal.h"
 
-  return EFI_SUCCESS;
-}
+extern BOOLEAN  mPerformanceMeasurementEnabled;

```

Comment:
why not move those extern to the common SmmPerformanceLibInternal.h

Reply:
With the common implementation moved there, they are now.